lib/runner/global-reporter.js

```diff
@@ -20,7 +20,7 @@ class GlobalReporter {
     };
   }
 
-  constructor(reporter, settings) {
+  constructor(reporter='junit', settings) {
```

Comment:
I'm not sure why would this be undefined but it should use the value from `lib/settings/defaults.js`.

```suggestion
  constructor(reporter = DefaultSettings.default_reporter, settings) {
```

Reply:
Fixed